dll/win32/shell32/dialogs/folder_options.cpp

```diff
@@ -266,10 +275,34 @@ ShowFolderOptionsDialog(HWND hWnd, HINSTANCE hInst)
     pinfo.pszIcon = MAKEINTRESOURCEW(IDI_SHELL_FOLDER_OPTIONS);
     pinfo.pszCaption = szOptions;
     pinfo.pfnCallback = PropSheetProc;
+    pinfo.nStartPage = (UINT)param;
 
     PropertySheetW(&pinfo);
 
     stub.DestroyWindow();
+    return 0;
+}
+
+VOID WINAPI 
+ShowFolderOptionsDialog(UINT Page, BOOL Async = false)
```

Comment:
```suggestion
ShowFolderOptionsDialog(UINT Page, BOOL Async = FALSE)
```

Reply:
For people's mental health, please keep consistency between bool/true/false and BOOL/ TRUE/FALSE.

Reply:
Also, C++ `true / false` is _**NOT**_ the same as Win32 `TRUE / FALSE`. The first one (C++) can be byte-long, while the latter are 4-bytes (dword) long; also nothing guarantees that the internal representation of the C++ constants is the same as the one of the Win32 ones.